### ISSUE TITLE
Verify EULA (End User License Agreement)

### ISSUE THREAD

**Issue by speqtr** [qwertycoin-org/qwcx#7]
Updated [LICENSE.txt](https://github.com/qwertycoin-org/qwcx/blob/master/LICENSE.txt) per @Xecute0101 request.

And since I'm not a lawyer, I'm opening this issue to discuss (and maybe update) `LICENSE.txt` content with the team 🙂

cc @exploshot @nnian @Xecute0101 @who-biz

**Comment by speqtr** [comment]
@Xecute0101 requirement for "splash screen" can be found under 2.3.

**Comment by who-biz** [comment]
I have left the organization, and need time to thoroughly review this, if it applies to my work.  For now, LMDB code is public and pretty much functional. I don’t foresee myself being willing to do anymore work beyond finishing that up, where necessary.

If someone could please let me know if this applies to LMDB or not, that would be great.

**Comment by Xecute0101** [comment]
> I have left the organization, and need time to thoroughly review this, if it applies to my work. For now, LMDB code is public and pretty much functional. I don’t foresee myself being willing to do anymore work beyond finishing that up, where necessary.
> 
> If someone could please let me know if this applies to LMDB or not, that would be great.

This is end user license agreement of wallet, so if you are a software user or a developer who will use the codebase of this particular repo will be subjected to EULA. Nothing to do with LMDB work since it is for daemon.

**Comment by nnian** [comment]
Okay i've read the terms. Looks like a default EULA with added 2.3 Slash Screen part?
So the "The Qwertycoin Team" is the new Slogans Tag for all development stuff?

**Comment by speqtr** [comment]
@nnian, yes, new EULA for QWCX wallet contains mostly standard terms with a small update.

**Comment by speqtr** [comment]
Regarding "legal name", I had in mind **The Qwertycoin Group**, **The Qwertycoin Team** and simply **Qwertycoin**. Decided to go with **The Qwertycoin Team**. If it's not right, please, let me know, we can change it to something else if needed.

@Xecute0101, what do you think about this? Which one should we use in your opinion?

**Comment by Xecute0101** [comment]
Qwertycoin Project or Qwertycoin Community, I think it is better as Qwertycoin Project in my opinion.

This part is not so critical as it is already uploaded. Please continue with the development.
